clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -64,6 +85,46 @@ const ScanResultsForm = () => {
 
   return (
     <Box maxW="full">
+      <AlertDialog
```

Comment:
Can this use the existing `ConfirmationModal` component instead?

Reply:
The buttons are different in the `ConfirmationModal`, but I could refactor that modal to take button text and an icon prop. Or maybe just make a `WarningModal` instead, but it would need to take `<Text>` components as children.